Assignment2/StockServiceFactoryTest.java

```diff
@@ -0,0 +1,56 @@
+package assignment2;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Alyson Hudak
+ */
+
+public class StockServiceFactoryTest 
+{
+ 
+    @Before
+    public void setUp() 
+    {
+    }
+
+    /**
+     * Positive Test of getStock method, of class StockServiceFactory.
+     */
+    @Test
+    public void testGetStockPositive() 
+    {
+        String company = "Tech";
+        StockService result = StockServiceFactory.getStock(company);
+        assertNotNull(result);
```

Comment:
when writing an assert use the format 
assertXXX"diagnostic message", test)
This will make trouble shooting failed tests easier,